### DISCUSSION TITLE
"Anchor" Display Block Questions

### DISCUSSION THREAD

**Discussion by gramian** [lowdefy/lowdefy#1800] [Questions and Help] [answered]
Dear All,
I have two questions about the [`Anchor`](https://docs.lowdefy.com/Anchor) display block:

1. Can you clarify the difference between the `href` and `url` properties? Particularly when to use which.

2. If the `disabled` property is `true`, the `onClick` event can still be triggered. Is this correct?

Thank you very much for helping

**Comment by Gervwyk** [comment, marked as answer]
Hi @gramian 

1. In Lowdefy we serialise url query and page route for you, so the when using links you can set `pageId` and `urlQuery` to link to something like, `/page?_id=abc&a=123` with ease, `href` overwrite this with the default `href` behaviour. 
2. Ahh. yes, when disable, `onClick` still works. perhaps, this should not be the case. Might be a minor breaking change if we change it in a next release. Thanks for pointing this out. 

You can see the implementation here:
https://github.com/lowdefy/lowdefy/blob/main/packages/client/src/createLinkComponent.js
